src/sql/QueryEditor/QueryEditorHeader.tsx

```diff
@@ -14,7 +14,7 @@ export interface Props<
   extraHeaderElementLeft?: JSX.Element;
   extraHeaderElementRight?: JSX.Element;
   enableRunButton: boolean;
-  cancel?: (target: TQuery) => void;
```

Comment:
just double checking this change is intentional? (assuming it is? just seems like a breaking change i think so wanted to double check)

Reply:
we need it to satisfy the api that we're using

Reply:
retried it and i ended up not need it!

Reply:
i take that back, i did need it to be defined but i made it not a breaking change.